lib/modules/versioning/ubuntu/index.ts

```diff
@@ -54,8 +66,31 @@ function isStable(version: string): boolean {
 
 // digestion of version
 
+function getDatedContainerImageCodename(version: string): null | string {
+  const groups = /^(\w+)-(\d{8})$/.exec(version);
```

Comment:
Please use named groups for better readability

Reply:
done!